warehouse/warehouse.go

```diff
@@ -625,6 +625,9 @@ func (wh *HandleT) createJobs(warehouse warehouseutils.WarehouseT) (err error) {
 		// If it is present do nothing else delete it
 		if _, inProgess := wh.isUploadJobInProgress(warehouse, uploadID); !inProgess {
 			wh.deleteWaitingUploadJob(uploadID)
+		} else {
+			// reset priority if the upload is already in progress
+			priority = 0
```

Comment:
Should we reset in the case of nonwaiting jobs as well? Because there also we don't need it.

Reply:
non waiting as in? anything that is in progress, is non waiting right?

Reply:
I mean for jobs which we exported for those also we don't need to do it.